utils/rpms/daos.spec

```diff
@@ -367,7 +367,7 @@ mv %{buildroot}/%{conf_dir}/bash_completion.d %{buildroot}/%{_sysconfdir}
 sed -i -e '1s/env //' %{buildroot}{%{daoshome}/TESTING/ftest/{cart/cart_logtest,config_file_gen,launch,slurm_setup,util/verify_perms}.py,%{_bindir}/daos_storage_estimator.py,%{_datarootdir}/daos/control/setup_spdk.sh}
 
 # shouldn't have source files in a non-devel RPM
-rm -f %{buildroot}%{daoshome}/TESTING/ftest/cart/{test_linkage.cpp,utest_{hlc,portnumber,swim}.c,wrap_cmocka.h}
+rm -f %{buildroot}%{daoshome}/TESTING/ftest/cart/{test_linkage.cpp,utest_{hlc,portnumber,protocol,swim}.c,wrap_cmocka.h}
```

Comment:
why not utest_*.c ? its fine as is, just curious:)

Reply:
no idea but I just thought I'd stick to what was previously done :) @brianjmurrell might know :)

Reply:
I have no objection to `utest_*.c`.  I was just being conservative an only removing what I knew should not be in there and not wanting to create some issue down the road with removing more than what should be there.